pyEpiabm/pyEpiabm/cell.py

```diff
@@ -0,0 +1,30 @@
+from .microcell import Microcell
+
+
+class Cell:
+    """Class representing a Cell (Subset of Population)
+    Collection of :class:`Microcell` s and :class:`Person` s
+    """
+    def __init__(self):
+        """Constructor Method
+        """
+        self.microcells = []
+        self.people = []
```

Comment:
```suggestion
        self.persons = []
```
and so forth. Not important but this would be slightly more consistent

Reply:
Okay so 'persons' is the list of individuals, and 'people' is the length of that list effectively (following your OED definition). I made this change last (in a separate commit) so it can be easily checked/reverted if you don't like it.